### PR TITLE
Quote with image component

### PR DESCRIPTION
**Pull request by spencerldixon**
### Trello card

https://trello.com/c/7hcN0Ksx/6623-phase-4-amend-quote-component-design-and-include-images

### Context

GIT users need to feel inspired to teach and reassured that teaching is a viable career for them

### Changes proposed in this pull request

Create a `QuoteWithImageComponent` that accepts a title, html safe text content, image, background color, and option to toggle quotation marks on or off.

### CODE REVIEW

**Review by emmaachesongray — DISMISSED** [review]
Looks great to me 👍

**Comment by jenhadfield-dfe** [comment]
Screenshot 2024-10-18 at 16 54 49

Hey @spencerldixon I noticed if use links or headings it adds extra space above or below them, is there a way to tell it to ignore the extra space?

also at certain sizes the links and headings appear a different size to the other text?

Screenshot 2024-10-18 at 16 56 33

**Comment by jenhadfield-dfe** [comment]
@spencerldixon also do you know which is the right section or div class to have them sit in, so that they have space above and below when used on a page? the one I've used has white space on the right and left and would be better without

File: app/views/content/explore-teaching-advisers/_article.html.erb
Comment (gemmadallmandfe) [inline app/views/content/explore-teaching-advisers/_article.html.erb:15]: Just a reminder that this change will need removing from this PR before it is merged @spencerldixon @jenhadfield-dfe !

File: config/images.yml
Comment (gemmadallmandfe) [inline config/images.yml:223]: Just a reminder that this change will need removing from this PR before it is merged @spencerldixon @jenhadfield-dfe !

In addition, the image should also be removed from `/images/content/blog` - if it is needed when we actually start using this component, it can be added back in (without the surname)

**Comment by jenhadfield-dfe** [comment]
going to add the guidance in a separate PR for this (as I've recently updated it and this PR is behind). Have tested in a markdown file and it works fine. All looks good to me now!

**Comment by gemmadallmandfe** [comment]
@jenhadfield-dfe @spencerldixon I'm very happy for you to merge this if Jen is happy design-wise - **as long as the changes to the `/explore-teaching-advisers` page and the images are backed out first!**

**Review by martyn-w — APPROVED** [review]
👍 LGTM

**Review by martyn-w — APPROVED** [review]
👍  LGTM

**Pull request merged**
